### PR TITLE
Add extra logging for Content Share

### PR DESCRIPTION
**Pull request by RidipDe**
**Description of changes:**
Adding extra logs

**Testing**

1. Have you successfully run `npm run build:release` locally?
Yes
2. How did you test these changes?
Manual
3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it?
Yes, meeting
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
No
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/contentsharecontroller/DefaultContentShareController.ts [hunk 1]
```diff
@@ -187,6 +188,7 @@ export default class DefaultContentShareController
             this.contentAudioVideo.configuration.credentials.externalUserId
           );
         }
+        this.contentAudioVideo.logger.info('Setting up the ContentShareDidStart observer');
```
Comment (ltrung) [inline src/contentsharecontroller/DefaultContentShareController.ts:191]: I don't think this add any value.

File: src/contentsharecontroller/ContentShareMediaStreamBroker.ts [hunk 2]
```diff
@@ -23,6 +23,7 @@ export default class ContentShareMediaStreamBroker implements MediaStreamBroker
 
   async acquireAudioInputStream(): Promise<MediaStream> {
     if (this._mediaStream.getAudioTracks().length === 0) {
+      this.logger.info('Synthesizing fake audio stream');
```
Comment (ltrung) [inline src/contentsharecontroller/ContentShareMediaStreamBroker.ts:26]: Should we not use the word fake? Maybe something like 'No audio stream available. Synthesize an audio stream'

Comment (simmkyu) [inline src/contentsharecontroller/ContentShareMediaStreamBroker.ts:26]: We use the term "empty audio."
https://github.com/aws/amazon-chime-sdk-js/blob/7d7d0cbebdfc1a11113001ca76f2b7eddf379d7a/src/devicecontroller/DefaultDeviceController.ts#L943-L945

How about the following message?
> Create an empty audio device for the content share

File: src/contentsharecontroller/DefaultContentShareController.ts [hunk 3]
```diff
@@ -165,6 +165,7 @@ export default class DefaultContentShareController
         }
         const stream = this.mediaStreamBroker.mediaStream;
         if (stream.getVideoTracks().length > 0) {
+          this.contentAudioVideo.logger.info('Stream has video tracks');
```
Comment (simmkyu) [inline src/contentsharecontroller/DefaultContentShareController.ts:168]: To distinguish from the regular audio-video logs, include the term "content share." e.g., `The content share stream has ${stream.getVideoTracks().length} video track(s)`

File: src/contentsharecontroller/DefaultContentShareController.ts [hunk 4]
```diff
@@ -164,6 +164,9 @@ export default class DefaultContentShareController
           return;
         }
         const stream = this.mediaStreamBroker.mediaStream;
+        this.contentAudioVideo.logger.info(
```
Comment (ltrung) [inline src/contentsharecontroller/DefaultContentShareController.ts:167]: This should be just this.logger.info

File: src/contentsharecontroller/ContentShareMediaStreamBroker.ts (hunk 2 above)
Comment (RidipDe) [reply src/contentsharecontroller/ContentShareMediaStreamBroker.ts]: Changed!!

File: src/contentsharecontroller/DefaultContentShareController.ts (hunk 3 above)
Comment (RidipDe) [reply src/contentsharecontroller/DefaultContentShareController.ts]: Added this log line.

File: src/contentsharecontroller/DefaultContentShareController.ts (hunk 4 above)
Comment (RidipDe) [reply src/contentsharecontroller/DefaultContentShareController.ts:167]: DefaultContentShareController.ts does not have direct reference to the logger.

**Review by simmkyu — APPROVED** [review]

**Pull request closed without merging**